### PR TITLE
[Doc Feature][Video capability in ios Teams][2322937]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f423bdbf03d19c24e6f4652ce95ad5ea74fea8ec/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmobile-camera-image-permissions.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/40c8a1c7-496e-42f8-a467-0e75ca7500de) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3aa499bd4a3893aba07d2c90cea46685957b5155/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmobile-camera-image-permissions.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fe44a93b-36e3-4bed-aa14-9cd42ca09fda) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/15147e2980054af1644c908e13e077e4ad48e63e/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmobile-camera-image-permissions.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f379fd80-ae8d-4bee-b648-be019128c9f1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3b3db33e1e595accee50d26092126a1da9a0d037/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmobile-camera-image-permissions.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4472c6d4-f096-4481-a594-498910867043) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4dc02eb1bfcb6214e6c9d5a5136491356bc924de/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmobile-camera-image-permissions.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bbb12c5c-79bf-4434-8893-d6e8e1b1035b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/11551986cba99025fd30eff70a92ebfeca1a63df/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 90 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5e186670-04ee-4b98-8531-7825bf61050a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bb5dcc58da26498c59dd33e68e466e165b3f6fc4/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 87 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bae3c9ea-fa0e-489d-b7ec-d77f9b57c0de) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c104d9c0a7666659bbd250d2a464dee5fe917e01/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 87 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ef8520fe-d080-41bf-a94a-639e38dd6a65) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1717adf23fe383953edca67f16342a54703d745e/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/83cbaf26-2f68-4a99-8d3d-6607415b2739) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ced8e7aa400ac3ef6ae0061536bfd37038c26846/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8aa46555-363f-4642-b3de-78d0b3a32306) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/device-capabilities/media-capabilities.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ ms.author: lajanuar
...
+For effective integration, you must have a good understanding of [code snippets](#code-snippets) for calling the respective APIs, which allow you to use native media capabilities. It's important to familiarize yourself with the [API response errors](#error-handling) to handle the errors in your Teams app.
 
 ## Advantages
 
-The main advantage of integrating device capabilities in your Teams apps is it leverages native Teams controls to provide a rich and immersive experience to your users. The following scenarios showcase the advantages of media capabilities:
+The main advantage of integrating device capabilities in your Teams apps is it uses native Teams controls to provide a rich and immersive experience to your users. The following scenarios showcase the advantages of media capabilities:
 
 * Allow the user to capture the rough mockups drawn on a physical whiteboard through the cell phone and use the captured images as poll options in Teams group chat.
```
Comment (reenudeswal) [inline msteams-platform/concepts/device-capabilities/media-capabilities.md:20]: we can update this section to include video related usecases also. eg: Students can do video reading assignments, capture photos and use as pdf to upload etc

File: msteams-platform/concepts/device-capabilities/media-capabilities.md [hunk 2]
```diff
@@ -137,6 +137,77 @@ microsoftTeams.media.selectMedia(mediaInput, (error: microsoftTeams.SdkError, at
...
+
+  * Capturing videos with `fullscreen: true`:
+
+       `fullscreen: true` opens the camera in video recording mode. It provides an option to use both front and rear camera and also provides other attributes as mentioned in the following example:
+
+       ```javascript
+        
+         const defaultLensVideoProps: microsoftTeams.media.VideoProps = {
+             sources: [microsoftTeams.media.Source.Camera, microsoftTeams.media.Source.Gallery],
+             startMode: microsoftTeams.media.CameraStartMode.Photo,
```
Comment (reenudeswal) [inline msteams-platform/concepts/device-capabilities/media-capabilities.md:150]: this should be startMode: microsoftTeams.media.CameraStartMode.Video

File: msteams-platform/concepts/device-capabilities/media-capabilities.md [hunk 3]
```diff
@@ -137,6 +137,77 @@ microsoftTeams.media.selectMedia(mediaInput, (error: microsoftTeams.SdkError, at
...
+             maxMediaCount: 6,
+             videoProps: defaultLensVideoProps
+        }
+       ```
+
+  * Capturing videos with `fullscreen: false`:
+
+       `fullscreen: false` opens the camera in video recording mode and uses the front camera only. Typically `fullscreen: false` is used when user wants to record video while reading content on the device screen.
+
+       This mode also supports `isStopButtonVisible: true`. This adds a stop button on the screen that allows user to stop the recording. If `isStopButtonVisible: false`, then it's expected that the recording is stopped by accepting an external selection to API.
```
Comment (reenudeswal) [inline msteams-platform/concepts/device-capabilities/media-capabilities.md:165]: When isStopButtonVisible =  false, recording can be stopped either by calling mediaController API or when the recording duration reached to  maxDuration time specified in video props. Also we should be adding a section for mediaController API(the api called to stop the video programatically.)

File: msteams-platform/concepts/device-capabilities/media-capabilities.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ Update your Teams app [manifest.json](../../resources/schema/manifest-schema.md#
 The [selectMedia](/javascript/api/@microsoft/teams-js/microsoftteams.media.media?view=msteams-client-js-latest&preserve-view=true), [getMedia](/javascript/api/@microsoft/teams-js/microsoftteams.media.mediachunk?view=msteams-client-js-latest&preserve-view=true), and [viewImages](/javascript/api/@microsoft/teams-js/microsoftteams.media.imageuri?view=msteams-client-js-latest&preserve-view=true) APIs enable you to use native media capabilities as follows:
 
 * Use the native **microphone** to allow users to **record audio** (record 10 minutes of conversation) from the device.
```
Comment (reenudeswal) [inline msteams-platform/concepts/device-capabilities/media-capabilities.md:46]: add similar line fro video mentioning that currently a video of maximum 5 minutes(300 seconds) can be recorded.

File: msteams-platform/concepts/device-capabilities/media-capabilities.md [hunk 5]
```diff
@@ -137,6 +137,77 @@ microsoftTeams.media.selectMedia(mediaInput, (error: microsoftTeams.SdkError, at
 });
 ```
 
+* Call `selectMedia` API for capturing videos using camera:
+
+  * Capturing videos with `fullscreen: true`:
```
Comment (reenudeswal) [inline msteams-platform/concepts/device-capabilities/media-capabilities.md:142]: `fullscreen: true` looks a bit confusing. The parameter name is different and this also looks like a drama name only. Can we  use something like : Capture videos in full Screen ? Just a suggestion, Mukul can confirm

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e17a3b10dd8670b260bdaeb6dc5298d760ec5bc4/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3baa3e57-9d2a-422c-a144-1581975f7ce5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/446f7574744823106e819a20f328e0f6abae9073/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eb9e92c3-9b72-4e1e-85a3-bf01f5c06a84) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/822ca06cca43278b23a755a1b59505ea0c7a57c0/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7aef7219-3e0b-493f-9171-e56bc7979162) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/00f43d05ea416e78e0448772c69447e7fb7fff15/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2f9ce07d-66f4-4d42-8a99-dcf17da130de) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/597ef98bccd9db91a7728d6221243e01b3f21f32/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/16a48f83-de70-4977-a1f3-6b489f815561) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ad65b3a8f9e9c0f5035b6d0792aa85c16cbd7d2/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a5e52c6d-c176-488e-8bd0-9452285d545c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/19ef6453c98d6517bf922f5704ec8c36178ca10c/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/58799caf-90cf-4f48-a057-b6479c5e6d9c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a0613d0dacc6ec7d2786f5b553306187ea843438/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8e2c8590-dbd9-402a-8f90-18b41a6ddbff) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a0613d0dacc6ec7d2786f5b553306187ea843438/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/614c0842-8f92-442e-a7ab-0f0c696a6c29) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by reenudeswal — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/device-capabilities/media-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a98bb18d1b75775e8d31550e6fc7ae818cc0f9e0/msteams-platform%2Fconcepts%2Fdevice-capabilities%2Fmedia-capabilities.md) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/425adc92-1b94-4d80-960b-45bc0cd4b724) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a98bb18d1b75775e8d31550e6fc7ae818cc0f9e0/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/32402d70-ec3d-401d-a58e-d60921fe42f3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**